src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java

```diff
@@ -24,6 +24,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         out.println("<li>Bearchitect</li>");
         out.println("<li>58050237 Chayapol Limanon</li>");
         out.println("<li>57050294 Pichavarit Panyokaew</li>");
+	out.println("<li>57050269 Patipon Taweechat</li>");
```

Comment:
please pull the upstream to merge with your code and adjust the alignment of code.